src/integration/saasquatchUpdatesController.ts

```diff
@@ -61,9 +61,9 @@ export class saasquatchUpdatesController{
             const createContactBody = {
                 "properties": basicInfoAndProgramShareLinks
             }
-            await this.hubApiModel.createObject("contacts", createContactBody);
-               
-            
+            await this.hubApiModel.createObject("contacts", createContactBody,20060618);
```

Comment:
Was this hardcoded hub_id left over from testing?

Reply:
also try catch

Reply:
The hardcoded values for this values are necessary until we have a mapping for tenant alias and hubid. The hardcoded values currently represent the team hubid.